composer/bin/test-reporter

```diff
@@ -1,13 +1,19 @@
 #!/usr/bin/env php
 <?php
 
-$files = array(
+namespace CodeClimate\PhpTestReporter;
+
+use CodeClimate\PhpTestReporter\Constants\Version;
+
+$files = [
     __DIR__ . '/../../vendor/autoload.php',
-    __DIR__ . '/../../../../autoload.php'
-);
+    __DIR__ . '/../../../../autoload.php',
+];
 
-foreach ($files as $file) {
-    if (file_exists($file)) {
+foreach ($files as $file)
+{
```

Comment:
Was going to ask the same, since you explicitly made this change elsewhere. Probably not picked up by the engine because it doesn't end in `.php`.